brozzler/ydl.py

```diff
@@ -32,9 +32,20 @@
 thread_local = threading.local()
 
 
-def should_ytdlp(page, site):
+def should_ytdlp(worker, site, page):
     # called only after we've passed needs_browsing() check
     if page.status_code != 200:
+        logging.info("skipping ytdlp: non-200 page status")
+        return False
+    if site.skip_ytdlp:
```

Comment:
I think we need to add the `skip_ytdlp` attribute to the Site class in `model.py`

Reply:
After some discussion, it turns out this is handled for us in the doublethink library here: https://github.com/internetarchive/doublethink/blob/1dc2e0395ca697115a38e562e81eb922577bea41/doublethink/orm.py#L124C5-L125C1

Reply:
nevertheless, I've added to `Site`'s `populate_defaults`